tiledb/api/src/array/attribute/mod.rs

```diff
@@ -520,7 +520,10 @@ impl<'ctx> Factory<'ctx> for AttributeData {
     }
 }
 
-#[cfg(feature = "proptest-strategies")]
+#[cfg(feature = "arrow")]
```

Comment:
Should we not add the same `any(test, ...)` for arrow?

Reply:
I'd say the difference is that the proptest strategies we need to do the testing of the other stuff, whereas arrow is just a thing we want to test.